iris-fitDataset/python/iris_data.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2018 Google LLC. All Rights Reserved.
```

Comment:
Similar to above: no need to fork this Python file.

Reply:
done